base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
```

Comment:
FYI: setting the `min-width` to 414 will leave out many devices.

Reply:
Hi Namir, the moment i find the time, iam going to change the entire sizes of my application and all elements.
I started with static values but will change in one run everything to ratio values to get the app full responsive.

Thx for letting me know. I was just prototyping a bit with my app at this state.